soda/scientific/soda/scientific/anomaly_detection/anomaly_detector.py

```diff
@@ -133,8 +133,8 @@ def _parse_historical_check_results(self, check_results: Dict[str, List[Dict[str
                 "Anomaly Detection for this check yet."
             )
             parsed_check_results = AnomalyHistoricalCheckResults(results=[AnomalyResult()])
-            _df_measurements = pd.DataFrame.from_dict(parsed_check_results.dict()["results"])
-            return _df_measurements
+            _df_check_results = pd.DataFrame.from_dict(parsed_check_results.dict()["results"])
```

Comment:
This change is outside of the PR scope, but it was confusing for me each time I look anomaly detection. So, I decided to change the name. We made a typo here.